src/universal/modules/userDashboard/components/UserSettings/UserSettings.js

```diff
@@ -33,6 +35,17 @@ const UserSettings = (props) => {
             {renderActivity(activity)}
           </div>
           <div className={css(styles.row)}>
+            <Avatar hasBadge={false} picture={picture} size="large"/>
```

Comment:
Clearly, we could make this interaction a lot purdier/better. Open question: now or in subsequent PR?